test/testdata/lsp/hover.rb

```diff
@@ -57,6 +75,15 @@ def baz(arg)
   def self.anotherFunc()
     [1, "hello"]
   end
+
+  # Tests return markdown output
+  sig {void}
+  def tests_return_markdown
+    # ^^^^^^^^^^^^^^^^^^^^^ hover: Tests return markdown output
```

Comment:
Add in an assertion for the "---" horizontal line break that we've added too.

Reply:
59bb8bb27